.github/workflows/llvm.yaml

```diff
@@ -0,0 +1,41 @@
+name: Exhaustive LLVM Build
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  doit:
+    name: 'Build & Test (${{matrix.version}})'
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['10.0.1', '11.1.0']
```

Comment:
But wait, if we expect 7 through 11 to work why is this new workflow only testing 10 and 11? ah yes, tune in soon for the next PR...